po/ka.po

```diff
@@ -0,0 +1,614 @@
+# Georgian translation for gnome-pomodoro
+# Copyright (C) 2024 gnome-pomodoro authors
+# This file is distributed under the same license as the gnome-pomodoro package.
+# Temuri Doghonadze <temuri.doghonadze@gmail.com>, 2024.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: gnome-pomodoro\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-03-31 13:18+0200\n"
+"PO-Revision-Date: 2024-06-27 05:53+0200\n"
+"Last-Translator: Temuri Doghonadze <temuri.doghonadze@gmail.com>\n"
+"Language-Team: Georgian <(nothing)>\n"
+"Language: ka\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
```

Comment:
Please change it to `"Plural-Forms: nplurals=1; plural=0;\n"`

Then, `msgstr[1]` entries can be removed.

Reply:
It's default for poedit, couldn't convince author to replace it with correct one.
He complained that he's not going to learn "my language" :D

Anyways, corrected now.